Formula/mongosh.rb

```diff
@@ -3,16 +3,15 @@
 class Mongosh < Formula
   desc "MongoDB Shell to connect, configure, query, and work with your MongoDB database"
   homepage "https://github.com/mongodb-js/mongosh#readme"
-  url "https://registry.npmjs.org/@mongosh/cli-repl/-/cli-repl-0.15.5.tgz"
-  sha256 "dbfbe9c5c571d19740f2118f0f023b38836be8ea2e49a1c9be402f108d2900e0"
+  url "https://registry.npmjs.org/@mongosh/cli-repl/-/cli-repl-0.15.6.tgz"
+  sha256 "32cb9c1a7453b829071d682a00fc03a86ed28bad1f21515349241c088e1c2ff9"
   license "Apache-2.0"
 
   bottle do
-    sha256                               arm64_big_sur: "14b22c6f20b67fc2274369bf14f680ece3b8f6b8b011eb656f8d1377bce77c13"
-    sha256                               big_sur:       "e5c381d7596650e45053cb53c5a50a3bdbf1a9fc938d79b429cca74663695828"
-    sha256                               catalina:      "73d9bd50115a34fd475921601120e2830285622dc791c441707fd20dd3b0556d"
-    sha256                               mojave:        "c457b7ba16b47849c6281e5de7fe006eca3703263d91a435e02a120921a4b8ad"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:  "219aafa45ad0a113f99622196114980fcc3dcddc66a988aa632ee53ac585f08c"
+    sha256 arm64_big_sur: "5b4377a3429a66afcb34f8a14119112c045f86889c8a667f656282c2044c7ac1"
+    sha256 big_sur:       "b0b2c5bf6df51288ffc542c91394880689e7aa36fc5ab97d8b3e08d9d8fe145b"
+    sha256 catalina:      "099214e38e1c1ba6007719f3ff956e56b1a8c954280b3e6883bdc472d7677fe0"
+    sha256 mojave:        "9c46c746cf8f65a8418cb39237471cd6338135e398f2ace965a6c580e1165951"
```

Comment:
Sorry to pings again @rose-m but it seems like there's a new issue this time 😅. The `bottle` block shouldn't be edited, and I wonder why it's being modified by the PR creation logic.

Reply:
Yeah looking into it right now... I'm very much confused as the hashes that the PR wants to create there are from 0.14.0 bottles!? (see https://github.com/Homebrew/homebrew-core/commit/b514859b970c725d34e1aff24fe0da4ee575f025)

Reply:
And these are the hashes that are still present in the `mongodb-js` fork - https://github.com/mongodb-js/homebrew-core/blob/master/Formula/mongosh.rb. So I wonder why the logic I tested locally does not work correctly here...

Reply:
The fix is now implemented and it was a silly oversight where it grabbed the content to compute the new formula from... https://github.com/mongodb-js/mongosh/commit/8443a604ca4898ae3269ef0694cd010f32fa5dbe#diff-91188111d4bd4195d993bd80d7c1409d12c1e2f513d768eb6b84f619bf6f1c78R8

This PR is now updated (the source branch has been re-created). Now GitHub renders the diff pretty strangely - I think changing the base and resetting it will cause that to be correct now. Or we just close and re-open the PR :)

Reply:
It's basically the equivalent of this PR now: https://github.com/rose-m/homebrew-core/pull/3

Reply:
Thanks again, I'll open a new PR with just the latest commit (which is what we need) since there's a bunch of other commits here. 😄